### ISSUE TITLE
Remove state dependencies in components.

### ISSUE THREAD

**Issue by brodavi** [ProjectDrawdown/global-research-platform#63]
It is difficult to implement stories/documentation in Storybook due to state dependencies in components. Remove state dependencies in components. Create pure components and bound components. ( @litenull to explain how to do this. Also, this ticket might be best broken into one ticket per component )

**Comment by becca-tiessen** [comment]
Hey I could take a stab at this one, are there specific example files that have been done already that I could reference?

**Comment by litenull** [comment]
@becca-tiessen Hi! There is no example yet, but i think you could do something like this:

Example of component with state:

    const Component = () => {
      const workbook = useWorkbookSelector();
      return (
        <Section> {workbook.name} </Section>
      )
    }

What we need to do:

    const Component = ({ name }) => {
      return (
       <Section> {name} </Section>
      )
    }

Basically de-couple the views from the state. Let me know which components you would like to start with!
